wordpress-mobile/WordPress-Android
# Fix deeplink verification

Fixes #16695

To test:
1. Generate a release signed APK. 
2. Install the app on your device.
3. Run the command `adb shell pm get-app-links org.wordpress.android`
4. Confirm the app links of the app are correctly verified.

## Regression Notes
1. Potential unintended areas of impact

5. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A
6. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
